Components/Modal.jsx

```diff
@@ -89,17 +86,19 @@ export default class Modal extends React.Component {
 				{this.props.open && (
 					<div
 						style={this.props.style && this.props.style.background}
-						className={this.props.customClassNames.background || Modal.defaultProps.customClassNames.background}
+						className={
+							this.props.customClassNames.background || Modal.defaultProps.customClassNames.background
+						}
 					>
 						<div
 							style={this.props.style && this.props.style.holder}
 							className={this.props.customClassNames.holder || Modal.defaultProps.customClassNames.holder}
 						>
 							<div
-								style={
-									this.props.style && this.props.style.modal
+								style={this.props.style && this.props.style.modal}
+								className={
+									this.props.customClassNames.modal || Modal.defaultProps.customClassNames.modal
```

Comment:
We need to check if this is really necessary: this.props.customClassNames.modal || Modal.defaultProps.customClassNames.modal
I think not as defaultProps exists for this purpose.